### ISSUE TITLE
Installation instructions

### ISSUE THREAD

**Issue by marcoag** [osrf/ros2_test_cases#1484]
Follow the official installation instructions for the platform. If instructions have not been published for the platform, use the instructions for Rolling Ridley.

For source build platforms, this includes the installation of dependencies and building the packages.

## Configuration

* OS: Ubuntu Noble 24.04
* Build type: Binary
* Chip set: ARM64
* DDS: CycloneDDS

## Links

* [Installation instructions](https://docs.ros.org/en/jazzy/Installation.html)

**Comment by knmcguire** [comment]
Currently working on this issue with a Rapsberry 5 with Ubuntu Noble. It's pretty fresh but I just have installed and deinstalled ROS2 through debian so I hope that the removal was as clean as it could.

**Comment by knmcguire** [comment]
Terminal 1:
```
bitcraze@bitcraze-RP5:~$ . ~/ros2_jazzy/ros2-linux/setup.bash
bitcraze@bitcraze-RP5:~$ export RMW_IMPLEMENTATION=rmw_cyclonedds_cpp
bitcraze@bitcraze-RP5:~$ ros2 run demo_nodes_cpp talker
[INFO] [1714736098.340363005] [talker]: Publishing: 'Hello World: 1'
....
```

Terminal 2:
```
bitcraze@bitcraze-RP5:~$ . ~/ros2_jazzy/ros2-linux/setup.bash
bitcraze@bitcraze-RP5:~$ export RMW_IMPLEMENTATION=rmw_cyclonedds_cpp
bitcraze@bitcraze-RP5:~$ ros2 run demo_nodes_cpp talker
[INFO] [1714736128.340716372] [listener]: I heard: [Hello World: 31]
....
```

**Seems to work!** ✅


_Some thoughts:_

I did install the cyclone dds through debian packages, which also installed a whole bunch of other packages that it depends on... so not sure if that is what I was supposed to do. I did notice from the installation instructions that it said
```
rosdep install --from-paths ~/ros2_jazzy/ros2-linux/share --ignore-src -y --skip-keys "cyclonedds fastcdr fastrtps iceoryx_binding_c rmw_connextdds rti-connext-dds-6.0.1 urdfdom_headers"
```

When I removed the cyclonedds from the skip keys, the standard tests like I did here above worked as well but it seemed to have grabbed the bindings for rolling and not jazzy

**Comment by samuel-ongzx** [comment]
Thanks for testing @knmcguire! Just tried testing this again since this issue its till open.

## Configuration
- Host: NVIDIA Jetson Nano (ARM64)
- OS: Docker `ghcr.io/sloretz/ros-testing:jazzy-desktop`
- DDS: `CycloneDDS`

## Process
- [x] Install works (`yes`) 
[binaries for arm64](https://github.com/ros2/ros2/releases) includes `librmw_cyclonedds_cpp.so`. No additional installs required.

## Verification
<img width="864" alt="image" src="https://github.com/osrf/ros2_test_cases/assets/16714436/a03d9251-65fe-42fd-a345-776986e6c761">

**Comment by clalancette** [comment]
> I did install the cyclone dds through debian packages, which also installed a whole bunch of other packages that it depends on... so not sure if that is what I was supposed to do.

Yep, that's correct.

> When I removed the cyclonedds from the skip keys, the standard tests like I did here above worked as well but it seemed to have grabbed the bindings for rolling and not jazzy

Oh, hm.  So you are saying it installed `ros-rolling-cyclonedds` via rosdep?  Did it install anything else?

**Comment by knmcguire** [comment]
ah no sorry I don't remember which other packages it installed. Let me try again tomorrow on the RP 5 and I can confirm.

But other than that, I just went out of the way to try this. If I just follow the instructions on the word it all works well so you can close this issue for that

**Comment by clalancette** [comment]
> But other than that, I just went out of the way to try this. If I just follow the instructions on the word it all works well so you can close this issue for that

Fantastic, thanks.  Will close for now, feel free to reopen or comment again if you find something wonky.
